etc/deploy/aws/aws-doco-script.sh

```diff
@@ -0,0 +1,279 @@
+    --attach-policy-arn arn:aws:iam::${ACCOUNTID}:policy/${EBS_CSI_DRIVER_POLICY_NAME} \
+    --approve \
+    --role-only \
+    --tags Key=Name,Value=${GLOBAL_TAG}
+
+# aws cloudformation get created stack name
+STACK_NAME=$(aws cloudformation describe-stacks | jq -r '.Stacks[0].StackName')
+
+CREATED_ROLE_NAME=$(aws cloudformation describe-stack-resources --query 'StackResources[0].PhysicalResourceId' --output text --stack-name ${STACK_NAME})
+
+eksctl create addon --name aws-ebs-csi-driver --cluster ${CLUSTER_NAME} --service-account-role-arn arn:aws:iam::${ACCOUNTID}:role/${CREATED_ROLE_NAME} --force
+
+# Get the cluster VPC ids
+CLUSTER_VPC_IDS=$(aws eks describe-cluster --name ${CLUSTER_NAME} \
+    | jq -r '.cluster.resourcesVpcConfig.securityGroupIds[]')
+
+# AWS describe cluster get vpc id
+CLUSTER_VPC_ID=$(aws eks describe-cluster --name ${CLUSTER_NAME} | jq -r '.cluster.resourcesVpcConfig.vpcId')
+
+# Get the cluster public subnet ids as an array
+CLUSTER_SUBNET_IDS=$(aws ec2 describe-subnets --filter Name=vpc-id,Values=${CLUSTER_VPC_ID} --query 'Subnets[?MapPublicIpOnLaunch==`true`].SubnetId' | jq -r '.[]')
+# aws cli Create a DB subnet group
+CREATED_SUBNET_GROUP=$(aws rds create-db-subnet-group --db-subnet-group-name ${DB_SUBNET_GROUP_NAME} --db-subnet-group-description "DB subnet group - public subnets" --subnet-ids $CLUSTER_SUBNET_IDS --tags Key=Name,Value=${GLOBAL_TAG})
+
+# AWS CLI Create postgresql rds instance
+CREATED_DB_INSTANCE=$(aws rds create-db-instance \
+    --db-instance-identifier ${POSTGRES_SQL_ID} \
+    --db-name ${POSTGRES_SQL_DB_NAME_1} \
+    --db-instance-class db.m6g.large \
+    --engine postgres \
+    --master-username ${POSTGRES_SQL_DB_USER_NAME} \
+    --master-user-password ${POSTGRES_SQL_DB_USER_PASSWORD} \
+    --storage-type io1 \
+    --iops 1500 \
+    --allocated-storage 100 \
+    --vpc-security-group-ids ${CLUSTER_VPC_IDS} \
+    --db-subnet-group-name ${DB_SUBNET_GROUP_NAME}  \
+    --publicly-accessible \
+    --tags Key=Name,Value=${GLOBAL_TAG} \
+    --no-multi-az)
+
+# Check if the postgresql rds instance is available
+aws rds wait db-instance-available --db-instance-identifier ${POSTGRES_SQL_ID}
+
+# Amazon aws cli expose postgresql port 5432 on VPC security group
+AUTHORIZED_RESPONSE=$(aws ec2 authorize-security-group-ingress --group-id ${CLUSTER_VPC_IDS} --protocol tcp --port 5432 --cidr 0.0.0.0/0)
+
+# Get the postgresql rds instance endpoint
+POSTGRES_SQL_ENDPOINT=$(aws rds describe-db-instances --db-instance-identifier ${POSTGRES_SQL_ID} | jq -r '.DBInstances[0].Endpoint.Address')
+
+# create a second database named "dex" in your RDS instance for Pachyderm's authentication service. 
+# database must be named dex
+# create a new user account and grant it full CRUD permissions to both pachyderm and dex databases.
+PGPASSWORD=$POSTGRES_SQL_DB_USER_PASSWORD psql -h ${POSTGRES_SQL_ENDPOINT} -U $POSTGRES_SQL_DB_USER_NAME $POSTGRES_SQL_DB_NAME_1 << EOF
+SELECT datname FROM pg_database;
```

Comment:
Can this all be done with the PostgreSQL CLI tools? If not, does it make sense to execute within a transaction?